web-docs/docs/_partial/_features.mdx

```diff
@@ -0,0 +1,15 @@
+### Expands upon the Ewon Flexy's Java API
+
+- Improves flexibility with existing functionality
+- Adds numerous packages with new functionality
+
+### Flexible usage from simple to complex
+
+- Features are designed to be used in a variety of ways
+- Utilities are provided to make common tasks easier
+- Classes are designed to be extensible
+
+### Easy to use
+
+- Simply add the dependency to your project and start using it
```

Comment:
```suggestion
- Simply add the dependency to your Maven project and start using it
```
This might be more clear in case people try to add this to an ant project.

Reply:
Changed